src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -1189,7 +1188,7 @@ bool SotaUptaneClient::putManifestSimple() {
   // does not send event, so it can be used as a subset of other steps
   auto manifest = AssembleManifest();
   if (hasPendingUpdates()) {
-    LOG_INFO << "An update is pending, putManifest is disabled";
+    LOG_DEBUG << "An update is pending, putManifest is disabled";
```

Comment:
thanks, that scared me for a while until i read the code :)

Reply:
haha, yes I've got that feedback a lot. Will try to think more about logging as an user interface :).

Reply:
This is fine, but is it worth expanding the message more, perhaps? "disabled" sounds off to me; perhaps "An update is pending. Skipping manifest upload until installation is complete." would be better?

Reply:
ok, reworked it again